src/Core/src/Handlers/RefreshView/RefreshViewHandler.Android.cs

```diff
@@ -9,6 +9,8 @@ protected override MauiSwipeRefreshLayout CreatePlatformView()
 			return new MauiSwipeRefreshLayout(Context);
 		}
 
+		public override bool NeedsContainer => true;
```

Comment:
Could you include a comment in code here explaining why is necessary? Would be useful for other contributors and reviewing the code in the future.